app/human-detector-app/components/DeleteCameraButton.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { TouchableOpacity, Alert } from 'react-native';
+import {  FontAwesome5 } from '@expo/vector-icons'; 
+import { styles } from '../src/styles';
+import { BackendContext } from '../contexts/backendContext';
+import { UserContext } from '../contexts/userContext';
+import BackendService from '../services/backendService';
+import User from '../classes/User';
+import Camera from '../classes/Camera';
+
+const showAlert = (groupId: string, cameraId: string, backendContext: BackendService, userContext: User, setCameras: (cameras:Camera[]) => void) => {
+  Alert.alert(
+    "Delete Camera",
+    "Would you like to delete this Camera?\nWARNING: This will delete all notifications tied to this Camera.",
+    [
+      {
+        text: "Cancel",
+        onPress: () => console.log("Cancel Pressed"),
+        style: "cancel"
+      },
+      {
+        text: "Delete",
+        onPress: async () => {
+          const response: number | null = await backendContext.deleteCameraAPI(groupId, cameraId);
+          if (response === 200) { // if backend deletion was successful, moves onto deleting it for the user.
+            const groupObj = userContext.getGroupFromId(groupId);
+            if (groupObj) {
+              const cameraObj = groupObj.getCameraFromId(cameraId);
+              if (cameraObj) { // "removes" the notifications and the camera.
+                cameraObj.notifications = [];
+                const groupIndex = userContext.groupList.indexOf(groupObj);
+                const camIndex = groupObj.cameras.indexOf(cameraObj);
+                groupObj.removeCameraFromGroup(camIndex);
```

Comment:
This function needs to be updated to remove the camera from the map in a UserContext as well which is kind of annoying but you can just add it here.  You only have to worry about this for cameras